frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -33,6 +33,8 @@ import TrackerRemovalHero from "./images/tracker-removal-hero.svg";
 import TrackerRemovalIcon from "./images/tracker-removal-icon.svg";
 import PremiumSwedenHero from "./images/premium-expansion-sweden-hero.svg";
 import PremiumSwedenIcon from "./images/premium-expansion-sweden-icon.svg";
+import premiumEuExpansionHero from "./images/eu-expansion-hero.svg";
```

Comment:
Nit, but the other imported images start with an uppercase, so probably best to either switch these to uppercase, or the other ones to lowercase as well. (I know we're not consistent about this throughout the codebase, but inside a single file would be nice.)